docs/apu_flashing_ipxe.md

```diff
@@ -0,0 +1,207 @@
+```
+dhcp net1
+set filename pxelinux.0
+set next-server 192.168.0.108
+chain tftp://${next-server}/${filename} 
+```
+
+> The `X` number in `netX` interface can be different depending on the connector 
+to which Ethernet is connected. If selected interface is connected to network
+information similar to the showed below should appear:
+```
+Configuring (net1 00:0d:b9:47:bb:e1).................. ok
+```
+> MAC address should be displayed. If there is no MAC address that means that
+there is no connection to the network for that interface.
+
+> IP placed next to `next-server` should be correct IP of used PXE server.
+
+6. After few seconds PXE server boot menu  should appear:
+
+```
+				 lqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqk
+				 x         PXE server boot menu          x
+				 tqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqu
+				 x Install                               x
+				 x Debian-netboot                        x
+				 x Voyage-netinst                        x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 mqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqj
+
+				        Press [Tab] to edit options
+```
+
+Select `Debian-netboot` because it's the only OS with installed `flashrom`
+available in PXE server boot menu when instruction is being written. After that
+OS booting should start. 
+
+Number of OSes can be increased in the future.
+
+> For legacy versions of coreboot PXE server boot menu is incorrectly 
+displayed. Every displayed character is doubled.
+
+7. When the following prompt appears:
+```
+pcengines login: 
+
+```
+Type `root` as login. Then next prompt should appear:
+```
+Password: 
+```
+Type `root` as password to finish logging process.
+
+8. Now you can start flashing process. To flash firmware with `flashrom` usage
+type:
+```
+flashrom -w <directory to ROM> -p internal
+```
+E.g.:
+```
+flashrom -w /root/coreboot.rom -p internal
+```
+
+After correct firmware flashing the following message should appear:
+```
+Reading old flash chip contents... done.
+Erasing and writing flash chip... Erase/write done.
+Verifying flash... VERIFIED.
+```
+9. Now you can reboot the platform.
+
+> Sometimes after APUx flashing platform doesn't turn on after warm boot. In 
+that situation cold boot is required.
+
+Sending ROM image to APUx device with `scp` usage
+-------------------------------------------------
+
+To send ROM image to device you can use `scp`.
+```
+cd <directory with ROM image>
+scp <ROM image> root@<IP of APUx to flash>:<directory to store ROM image on APUx>
+```
+E.g.:
+```
+cd /home/me/coreboot/build
+scp coreboot.rom root@192.168.0.123:/root
```

Comment:
It will be safer to use `/tmp` directory. In the case of voyage linux, rootfs is mounted as read-only by default.

Reply:
`/root` directories in instructions changed to `/tmp`